### PR TITLE
Tournament mode!

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: tools/shared/pilot.c [hunk 1]
```diff
@@ -159,7 +159,21 @@ void print_pilot_info(sd_pilot *pilot) {
         printf("  - Unk. Float B:    %f\n", pilot->unk_f_b);
 
         printf("  - Palette:\n");
-        print_bytes((char *)pilot->palette.data, 144, 16, 4);
+
+        printf("    ");
+        for(int i = 0; i < 48; i+=1) {
+            uint8_t r = pilot->palette.data[i+1][0];
+            uint8_t g = pilot->palette.data[i+1][1];
+            uint8_t b = pilot->palette.data[i+2][2];
```
Comment (katajakasa) [inline tools/shared/pilot.c:167]: data[] +1, +1, +2 ?

File: src/game/gui/pilotpic.c [hunk 2]
```diff
@@ -52,27 +43,45 @@ void pilotpic_select(component *c, int pic_id, int pilot_id) {
...
+    sprite_create(local->img, &spr, -1);
 
     // Position and size hints for the gui component
     // These are set on layout function call
     component_set_size_hints(c, local->img->data->w, local->img->data->h);
 
     // Save some information
     local->selected = pilot_id;
-    local->max = pics.photo_count;
+    local->max = 4; // TODO pics.photo_count;
```
Comment (katajakasa) [inline src/game/gui/pilotpic.c:83]: photo_count didn't work ?

File: src/game/gui/trnselect.c [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    if(g->label) {
+        component_render(g->label);
+    }
+}
+
+void load_description(component **c, const char *desc) {
+    int width = 320, center = 0, vmove = 0, size = -1, color = -1, x = 0;
+    const char *end = desc;
+    char *ptr = NULL;
+    // don't judge me!
```
Comment (katajakasa) [inline src/game/gui/trnselect.c:38]: We might want to split this parser to it's own thing later on. I think there were some other places that used this also?

Comment (Vagabond) [reply src/game/gui/trnselect.c:38]: Yes, the help/about stuff also uses it

File: src/game/gui/pilotpic.c (hunk 2 above)
Comment (Vagabond) [reply src/game/gui/pilotpic.c:83]: this whole widget needs a rewrite to keep the pic file in memory until its freed

**Review by katajakasa — APPROVED** [review]
It's fiiiiine.

**Review by katajakasa — APPROVED** [review]

**Pull request merged**